crates/rpc/rpc-api/src/otterscan.rs

```diff
@@ -24,7 +24,7 @@ pub trait Otterscan {
 
     /// Check if a certain address contains a deployed code.
     #[method(name = "hasCode")]
-    async fn has_code(&self, address: Address, block_number: Option<u64>) -> RpcResult<bool>;
+    async fn has_code(&self, address: Address, block_id: Option<BlockId>) -> RpcResult<bool>;
```

Comment:
this is incorrect, this makes it incompatible with the otterscan API this must be u64 not hex

Reply:
okay this is actually no longer the case, so BlockId is correct now

https://otterscan.github.io/execution-apis/api-documentation/